lib/uvobot/notifications.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'notifications/notifier'
```

Comment:
Moze. Pouzivam to na skupinovy require, ked rozpisem jednotlive notifiers nebude ho treba.